manticore/core/manticore.py

```diff
@@ -355,8 +369,60 @@ def __init__(self, initial_state, workspace_url=None, outputspace_url=None, **kw
         self._put_state(initial_state)
 
         # Workers will use manticore __dict__ So lets spawn them last
+        self._is_main = None
```

Comment:
Where is this used?

Reply:
Nowhere that I'm aware of, or can find with a quick grepping